### DISCUSSION TITLE
关于作业hw7-ordering的一点小疑问

### DISCUSSION THREAD

**Discussion by Martin7-1** [courses-at-nju-by-hfwei/discrete-math-problem-sets#5] [Q&A] [answered]
老师请问一下：查了几本参考书都说拟序是反自反的...是对的吗

**Comment by hengxin** [comment, marked as answer]
这是 “拟序” 与 “严格拟序” 之间的区别，类似于 “偏序” 与 “严格偏序” 之间的区别。但是，很多书籍都不会明确地说明自己指的到底是严格的版本还是非严格的版本。参见 wiki 页面: https://en.wikipedia.org/wiki/Preorder#Formal_definition。

总结：拟序，要求自反与传递；严格拟序，要求反自反与传递。没错，严格拟序与严格偏序是一样的。不过，严格偏序比严格拟序更常用。

**Reply by Martin7-1** [reply]
好的，谢谢老师

**Reply by hengxin** [reply]
@Martin7-1 多加了一点解释
